src/ChrCopyNumber.cpp

```diff
@@ -562,12 +588,14 @@ void ChrCopyNumber::recalculateRatio(double *a, int degree) {
                 //use a threshold, but correct using notN profile
                 if (mappabilityProfile_.size()>0) {
                     if ((x>0)&&(mappabilityProfile_[i]>minMappabilityPerWindow)) //if ((x>0)&&(notNprofile_[i]!=0))
-                        ratio_[i] = ratio_[i]/polynomial(x,a,1,degree);
+                        //ratio_[i] = ratio_[i]/polynomial(x,a,1,degree);
+						ratio_[i] = ratio_[i]/1;
```

Comment:
@ilyasoifer this is the GC normalization disable. not sure if this is the best implementation. I can try adding a flag I guess...

Reply:
see my comment about using polynomial for GCprofile